### PR TITLE
Ui suggestion fixes

### PR DESCRIPTION
**Pull request by ejohnson44**
Some changes to the UI based on suggestions and some of my own things:
- opens to menu fragment now, instead of just the blank playlist or the network
- playbar colors changed a bit, and added a border between it and the other things
- removed lines between connected users
- shifted the song title things over some
- made the playbar autoscroll when you tap it if the song is too long to fit in the view

### CODE REVIEW

File: SoundStream/src/com/lastcrusade/soundstream/util/Transitions.java [hunk 1]
```diff
@@ -46,10 +46,11 @@
 
     //Home is where you get sent after connecting to the network - for now
```
Comment (bryansills) [inline SoundStream/src/com/lastcrusade/soundstream/util/Transitions.java]: Fix this comment

**Comment by bryansills** [comment]
Aren't there multiple placeholder images in the repo? If so, can we remove them?

Also can we get some screenshots of the changes?

**Comment by ejohnson44** [comment]
What do you mean by placeholder images?

**Comment by ejohnson44** [comment]
New song view/playbar
playbarAnimation

new landing page
onstart

File: SoundStream/res/layout/fragment_playbar.xml [hunk 2]
```diff
@@ -20,15 +20,22 @@
...
+        android:layout_width="match_parent"
+        android:background="@color/light_gray"
+        android:alpha=".5"/>
    <!-- singleLine is deprecated, but it appears to be
         the only way to get the fading to work correctly.
         All of the advise found online suggests to just
         use the deprecated method -->
+    <!--focusable and scrolling attributes make the song
+    	title scroll when clicked. There appears to be no
+    	default way to set the speed - revisit later.   -->
```
Comment (reidbaker) [inline SoundStream/res/layout/fragment_playbar.xml:37]: I think we shoul make a trello card for this

File: SoundStream/src/com/lastcrusade/soundstream/CoreActivity.java [hunk 3]
```diff
@@ -133,9 +135,14 @@ public void enableSlidingMenu(){
     public void showPlaybar(){
         getSupportFragmentManager()
         .beginTransaction()
-        .replace(R.id.playbar, playbar)
+        .show(playbar)
+        .commit();
+    }
+    
+    public void hidePlaybar(){
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/CoreActivity.java:142]: YEA! :partly_sunny:

File: grail-diary [hunk 4]
```diff
@@ -0,0 +1 @@
```
Comment (reidbaker) [inline grail-diary]: This is bad. You don't want to check in any sub module changes.

Comment (ejohnson44) [inline grail-diary]: How do I get rid of it? I didn't realize that this got in there

File: SoundStream/res/layout/fragment_playbar.xml (hunk 2 above)
Comment (ejohnson44) [inline SoundStream/res/layout/fragment_playbar.xml:37]: Done

File: grail-diary (hunk 4 above)
Comment (reidbaker) [inline grail-diary]: I am trying to figure out that now. It should not have been possible for you to have this as a sub module. It was part of the git ignore.

**Comment by reidbaker** [comment]
This looks good to pull in. :rocket:

File: SoundStream/res/values/colors.xml [hunk 5]
```diff
@@ -29,4 +29,5 @@
     <color name="dark_gray">#333333</color>
     <color name="light_gray">#BBBBBB</color>
     <color name="white">#FFFFFF</color>
+    <color name="playbar">#DEDEDE</color>
```
Comment (dgreenhalgh) [inline SoundStream/res/values/colors.xml:32]: http://images1.wikia.nocookie.net/__cb20110825044212/kirby/en/images/thumb/b/bd/Dedede_KRTDL.png/250px-Dedede_KRTDL.png

**Comment by dgreenhalgh** [comment]
:shipit:

**Pull request merged**